src/Workspaces/CSharpTest/RegularExpressions/CSharpRegexParserTests_TestGeneration.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#if false
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+using Microsoft.CodeAnalysis.CSharp.RegularExpressions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.RegularExpressions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.RegularExpressions
+{
+    internal class LogicalStringComparer : IComparer<string>
```

Comment:
Useful code i used to quickly generate tons of test cases from lots of .net regex examples i found.  Can be removed from final checkin.